packages/ToggleSwitch/ToggleSwitch.md

```diff
@@ -1,3 +1,46 @@
+### Usage Criteria
+
+- `ToggleSwitch` button is a controlled component that requires an `onClick` handler that will trigger `true` or `false` value of `aria-checked` attribute.
```

Comment:
Please remove periods from the end of lists.

Reply:
Done